packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -86,8 +64,13 @@ const StyledListItem = styled.li`
   }
 `;
 
-const List = ({ links }) => (
-  <StyledList role="list">
+const List = ({ links, trustProjectLink }) => (
+  <StyledList role="list" trustProjectLink={trustProjectLink} links={links}>
+    {trustProjectLink && (
+      <StyledListItem key={trustProjectLink.text} role="listitem">
```

Comment:
You could try to pass the `trustProjectLink` prop to the `StyledListItem` and then add the styles for the first link inside the `StyledListItem`:

```
const StyledListItem = styled.li`
  min-width: 50%;

  ${({ trustProjectLink }) =>
    trustProjectLink &&
    `
      border-bottom: 1px solid ${C_SHADOW};
      padding: ${GEL_SPACING} 0;
      margin-bottom: ${GEL_SPACING};
      grid-column: 1/-1;
      @supports not (${grid}) {
        width: 100%;
      }
    `}

  @supports not (${grid}) {
    display: inline-block;
  }
`;

Reply:
The logic seems correct, but it's not working for me locally. Is it working for anyone else?